rclrs/src/parameter.rs

```diff
@@ -776,13 +776,12 @@ pub(crate) struct ParameterInterface {
 
 impl ParameterInterface {
     pub(crate) fn new(
-        rcl_node_mtx: &Arc<Mutex<rcl_node_t>>,
+        rcl_node: &rcl_node_t,
         node_arguments: &rcl_arguments_t,
         global_arguments: &rcl_arguments_t,
     ) -> Result<Self, RclrsError> {
-        let rcl_node = rcl_node_mtx.lock().unwrap();
         let override_map = unsafe {
-            let fqn = call_string_getter_with_handle(&rcl_node, rcl_node_get_fully_qualified_name);
+            let fqn = call_string_getter_with_handle(rcl_node, rcl_node_get_fully_qualified_name);
```

Comment:
This one is a bit tricky because the terminology in both cases is correct. The `rcl_node_t` is indeed a handle for a node provided by `rcl`. The `NodeHandle` struct is simply the `rclrs` struct that's responsible for ensuring that the use of the `rcl_node_t` handle is safe.

We could call it `call_string_getter_with_rcl_node` 🤷‍♂️

Reply:
Renamed: [cf0d434](https://github.com/ros2-rust/ros2_rust/pull/386/commits/cf0d434814a7c02331478268f7685b664aa3d69d)